eks_infrastructure/patch_hosts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#/ Usage: ./patch_hosts.sh
+
+set -ex
+source ./helper.sh
+
+# Parse parameters from build_param.json config file
+EKS_CLUSTERS=($(jq -r '.eks_clusters[]' build_param.json))
```

Comment:
Can we use a separate template similar to build_param.json file that includes the upgrade operation and all the clusters. This file is not supposed to be changed, specially the operation field

The build_param.json is meant to be changed for different operations, in which case this upgrade job will not serve its purpose.

Reply:
Will do.